Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "elf2tab"
-version = "0.7.0-dev"
+version = "0.8.0-dev"
```

Comment:
```suggestion
version = "0.7.0-dev"
```

Reply:
Shouldn't we increase the version due to this new header?

Reply:
0.7 is already an increase.

Reply:
Got it now :)